remote-api/src/main/java/org/alfresco/rest/api/nodes/NodeRuleSetLinksRelation.java

```diff
@@ -67,6 +69,24 @@ public List<RuleSetLink> create(String nodeId, List<RuleSetLink> ruleSetLinksBod
                 .collect(Collectors.toList());
     }
 
+    /**
+     * Remove link between a rule set and a folder rule for given rule set's and folder's node IDs.
+     * <p>
+     * - DELETE /nodes/{folderNodeId}/rule-set-links/{rulesetId}
```

Comment:
```suggestion
     * - DELETE /nodes/{folderNodeId}/rule-set-links/{ruleSetId}
```

Reply:
Not sure what the suggestion is here

Reply:
:joy: You can just accept it :yum: 

I've suggested changing a lowercase `s` to an uppercase `S`.